src/main/views/subscription-management.njk

```diff
@@ -120,4 +120,18 @@
   {{ super() }}
   <script type="text/javascript" src="/assets/js/jquery.js"></script>
   <script type="text/javascript" src="/assets/js/mojAll.js"></script>
+
+  <script>
+    window.onload = function() {
```

Comment:
Is there a way we could do the below using Nunjucks instead of JS? Where possible I think it's better to render it via the server side instead of the frontend JS